mobile/test/common/integration/base_client_integration_test.h

```diff
@@ -61,17 +63,24 @@ class BaseClientIntegrationTest : public BaseIntegrationTest {
   envoyToMobileHeaders(const Http::TestRequestHeaderMapImpl& request_headers);
   Event::ProvisionalDispatcherPtr dispatcher_ = std::make_unique<Event::ProvisionalDispatcher>();
   envoy_http_callbacks bridge_callbacks_;
-  ConditionalInitializer terminal_callback_;
-  callbacks_called cc_{0, 0, 0, 0, 0, 0, 0, "", &terminal_callback_, {}};
+  ConditionalInitializer* terminal_callback_;
+  std::vector<std::unique_ptr<ConditionalInitializer>> multi_terminal_callbacks_;
+  callbacks_called* cc_;
+  std::vector<std::unique_ptr<callbacks_called>> multi_cc_;
   Http::TestRequestHeaderMapImpl default_request_headers_;
   Event::DispatcherPtr full_dispatcher_;
   Platform::StreamPrototypeSharedPtr stream_prototype_;
+  std::vector<Platform::StreamPrototypeSharedPtr> multi_stream_prototypes_;
   Platform::StreamSharedPtr stream_;
+  std::vector<Platform::StreamSharedPtr> multi_streams_;
```

Comment:
`per_engine_stream` or something like that? Not immediately clear that this is one active stream per engine